.github/workflows/agw-build-publish-container.yml

```diff
@@ -31,9 +31,7 @@ jobs:
 
   build-containers:
     outputs:
-      digest_c: ${{ steps.docker-builder-c.outputs.digest }}
-      digest_python: ${{ steps.docker-builder-python.outputs.digest }}
-      digest_go: ${{ steps.docker-builder-go.outputs.digest }}
+      sha: ${{ steps.get-short-git-sha.outputs.sha }}
```

Comment:
The images have an 8 char sha tag - this sha is longer. Is the registry really working with the long hash?

Reply:
The `sha`-variable is the shortened 8-char hash, line 59.